### DISCUSSION TITLE
Dialect translations

### DISCUSSION THREAD

**Discussion by mpfl** [TomBursch/kitchenowl#295] [Q&A]
I speak German (very well) and English (native speaker) and notice that some of the English translations have a certain Denglisch feel to them.

Unfortunately, US, UK, and Australian English all differ in how we refer to certain groceries (mince beef vs ground beef, eggplant vs aubergine, courgette vs zucchini). I'm confident suggesting Australian English translations, but would not be sure if those products are referred to that way in US or UK English.

Unfortunately Weblate wouldn't let me add an en_AU translation.

How should I proceed? Should I make en_au suggestions for the en translations, or would you like to create an en_au translation for me to work on?

**Comment by TomBursch** [comment]
Strange that it didn't let you do that. I have created the Australian English components in Weblate for you to edit. But You are also welcome to edit the base translation if something seems particularly odd.
